vent/core/rq-worker/file_watch.py

```diff
@@ -120,22 +119,23 @@ def file_queue(path, template_path="/vent/"):
         volumes = {path: {'bind': path, 'mode': 'ro'}}
 
         # start containers
-        for image in images:
-            # TODO check for availability of gpu(s),
-            #      otherwise queue it up until it's
-            #      available
-            if 'volumes' in configs[image]:
-                for volume in volumes:
-                    configs[image]['volumes'][volume] = volumes[volume]
-            else:
-                configs[image]['volumes'] = volumes
-            d_client.containers.run(image=image,
-                                    command=path,
-                                    labels=labels,
-                                    detach=True,
-                                    log_config=log_config,
-                                    **configs[image])
-        status = (True, images)
+        if status[0]:
```

Comment:
yeah, some of them are (particularly the cores) but if they are, and some other container failed that they depend on they'll fail too, and if they're not they should still run.